Kconfig

```diff
@@ -197,6 +197,10 @@ menu "LVGL configuration"
 				> 1 requires an operating system enabled in `LV_USE_OS`
 				> 1 means multiply threads will render the screen in parallel
 
+		config LV_DRAW_THREAD_STACKSIZE
+			int "stack size of draw thread"
+			default 32768
```

Comment:
config LV_DRAW_THREAD_STACKSIZE
			int "stack size of draw thread"
			default 32768
                        depends on LV_USE_OS > 0

Reply:
done